tools/oot-driver/playbooks/deploy_sro.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Play to deploy SRO on a HUB cluster
```

Comment:
you also need the nfd to be deploy.

please check the feature-configs/deploy/sro/ folder

Reply:
1. NFD is automatically deployed as a pre-requisite to SRO.
2. We do not actually need NFD on the HUB cluster since we don't want it to label the HUB cluster itself as we install SRO on the HUB.  For now, we let SRO deploy NFD as a pre-requisite but we do not create an instance of the CR because we don't want the HUB labeled.

Reply:
Seems that SRO deploys NFD automatically when I add the SRO Subscription.

Reply:
You need to deploy the NFD special resource or else SRO wont stay running. This [file](https://github.com/openshift-kni/cnf-features-deploy/blob/master/feature-configs/deploy/sro/nfd-config.yaml)